core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -97,6 +97,49 @@ package object config {
     .toSequence
     .createWithDefault(Nil)
 
+  // Blacklist confs
+  private[spark] val BLACKLIST_ENABLED =
+    ConfigBuilder("spark.scheduler.blacklist.enabled")
+    .booleanConf
+    .createOptional
+
+  private[spark] val MAX_TASK_ATTEMPTS_PER_NODE =
+    ConfigBuilder("spark.blacklist.maxTaskAttemptsPerNode")
```

Comment:
Can you change "Node" here and below to be "Worker"? Better to avoid extra terminology here I think, since the executor / worker differentiation is already confusing.